### DISCUSSION TITLE
Is it possible to customize the tool's core without losing compatibility with future versions?

### DISCUSSION THREAD

**Discussion by renanaquinno** [Budibase/budibase#12408] [Help] [answered]
Is it possible to customize the tool's core without losing compatibility with future versions? Ex: changing the RBAC permissions system

**Comment by ConorWebb96** [comment, marked as answer]
Hey @renanaquinno,

It would likely be difficult as you'd have to pull and merge the latest changes. Any conflicts you'd then have to fix on your own. It is doable, but you probably wouldn't have much support for when you run into issues relating to the changes you've made to the core product.

I hope this makes sense.
